auth/auth.js

```diff
@@ -30,10 +31,18 @@ async function generateToken(email, clientdata) {
 }
 
 async function generateUser(userDetails) {
+  await bcrypt.hashSync(userDetails.password, SECRET, (err, hash) => {
+    userDetails.password = hash;
+  });
   let resultdata = axios.post(clients.POST.clients, userDetails);
```

Comment:
Axios is promise based, always call axios with await otherwise you'll get pending promises.

Reply:
done

Reply:
done